Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -833,6 +833,17 @@ theorem mk_subtype_mono {p q : α → Prop} (h : ∀ x, p x → q x) :
     #{ x // p x } ≤ #{ x // q x } :=
   ⟨embeddingOfSubset _ _ h⟩
 
+lemma card_ssubset' {A B : Set α} {hfin : #B < ℵ₀} (hlt : A < B) : #A < #B := by
```

Comment:
Unless something has changed in the past few months I believe the preferred form is this:
```suggestion
lemma card_ssubset' {A B : Set α} {hfin : #B < ℵ₀} (hlt : A ⊂ B) : #A < #B := by
```
Also, `hfin` should be explicit rather than implicit.